articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,493 @@
+
+> [!Note]
+> For domain computers, you can use a GPO to do this.
+
+#### Linux Machines
+
+* Ensure that there’s network connectivity between the Linux computer and the process server. You can test this by pinging the configuration server from the Linux machine.
+
+* On the machine for which you enable replication, you need an account that's a root user on the source Linux server:
+
+  * You specify this account when you set up Site Recovery. Then the process server uses this account to install the Mobility service when replication is enabled.
+
+  * This account will only be used by Site Recovery for the push installation, and to update the Mobility service.
+
+* Check that the /etc/hosts file on the source Linux server has entries that map the local hostname to IP addresses associated with all network adapters.
+
+* Install the latest openssh, openssh-server, and openssl packages on the computer that you want to replicate.
+
+* Ensure that Secure Shell (SSH) is enabled and running on port 22.
+
+* Enable SFTP subsystem and password authentication in the sshd_config file. To do this:
+
+  1. Sign in as root.
+
+  2. Find the line that begins with PasswordAuthentication, in the /etc/ssh/sshd_config file. Uncomment the line and change the value to yes.
+
+  3. Find the line that begins with Subsystem and uncomment the line.
+
+  4. Restart the sshd service.
+
+### Note down the VMs private IP address
+
+For every machine you want to replicate you will need to find and note down the IP address of the machine:
+
+1. In the Azure Stack Portal, find the VM you want to replicate.
+
+2. From the Resource menu, select **Networking**.
+
+3. Note down the private IP address.
+
+    ![Azure Stack private IP](images/azs-browser-networking-private-ip.png)
+
+## Step 2. Creating a new vault
+
+1. In the Public Azure portal navigate to **Create a resource** > **Management Tools** > **Backup and Site Recovery (OMS)**.
+
+2. In **Name** enter the name you want to call the vault.
+
+3. In **Resource group**, create or select a resource group.
+
+4. In **Location** choose a region.
+
+5. Click **OK** and the vault will be deployed.
+
+    ![Azure Stack recovery vault](images/azs-browser-recovery-vault.png)
+
+## Step 3. Setting a replication goal
+
+1. On the **Recovery Services vaults** tab on Public Azure, select the vault you have created.
+
+2. Select the **Site Recovery** tab under **Getting Started**.
+
+3. Select **Prepare Infrastructure**.
+
+4. In the *Protection goal* blade, from the **Where are your machines located?** list, select **On-premises**.
+
+5. From the **Where do you want to replicate your machines** list, select **To Azure**.
+
+6. From the **Are your machines virtualized** list, select **Not virtualized / Other**, then click **OK**.
+
+    ![Azure Stack recovery vault configuration](images/azs-browser-recovery-vault-configuration.png)
+
+7. In the *Deployment Planning* blade select the **Yes, I have done it option**.
```

Comment:
In the *Deployment Planning* blade, select **Yes, I have done it**.

Reply:
done